test/Python/numpy_conversion.swift

```diff
@@ -35,6 +35,17 @@ NumpyConversionTests.test("array-conversion") {
   let numpyArray2D = np.ones([2, 3])
   expectNil(Array<Float>(numpy: numpyArray2D))
 
+  let reshaped = np.reshape(numpyArray2D, 6)
+  if let array = expectNotNil(Array<Double>(numpy: reshaped)) {
+    expectEqual([1.0, 1.0, 1.0, 1.0, 1.0, 1.0], array)
+  }
+
+  let numpyArray1D = np.ones(28)
+  let reshaped3D = np.reshape(numpyArray1D, [2, 7, 2])
```

Comment:
Sorry, i meant to ask you to add reshape tests to https://github.com/apple/swift/blob/4e84b9267991c2e88342c83e8698914e335a9e44/test/TensorFlowRuntime/numpy_conversion.swift, where `TensorShape` is available.